tbprofiler/6.4.1/Dockerfile

```diff
@@ -0,0 +1,66 @@
+FROM mambaorg/micromamba:1.5.8 AS app
+
+USER root
+WORKDIR /
+
+ARG TBPROFILER_VER="6.4.1"
+
+# this version is the shortened commit hash on the `master` branch here https://github.com/jodyphelan/tbdb/
+# commits are found on https://github.com/jodyphelan/tbdb/commits/master
+# this was the latest commit as of 2024-10-31
+ARG TBDB_COMMIT="2c92475219416a449e89782f2b768149d26f7979"
+
+# LABEL instructions tag the image with metadata that might be important to the user
+LABEL base.image="micromamba:1.5.8"
+LABEL dockerfile.version="1"
+LABEL software="tbprofiler"
+LABEL software.version="${TBPROFILER_VER}"
+LABEL description="The pipeline aligns reads to the H37Rv reference using bowtie2, BWA or minimap2 and then calls variants using bcftools. These variants are then compared to a drug-resistance database."
+LABEL website="https://github.com/jodyphelan/TBProfiler/"
+LABEL license="https://github.com/jodyphelan/TBProfiler/blob/master/LICENSE"
+LABEL maintainer="John Arnn"
+LABEL maintainer.email="jarnn@utah.gov"
+LABEL maintainer2="Curtis Kapsak"
+LABEL maintainer2.email="kapsakcj@gmail.com"
+LABEL maintainer3="Sage Wright"
+LABEL maintainer3.email="sagemwright@gmail.com"
+
+# Install dependencies via apt-get; cleanup apt garbage
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    wget \
+    ca-certificates \
+    procps && \
+    apt-get autoclean && rm -rf /var/lib/apt/lists/*
+
+# install tb-profiler via bioconda; install into 'base' conda env
+RUN micromamba install --yes --name base --channel conda-forge --channel bioconda  \
+    tb-profiler=${TBPROFILER_VER} delly=1.2.6 && \
```

Comment:
I think we can remove `delly=1.2.6` now that the pathogen-profiler recipe pins delly 1.2.6: 

https://github.com/bioconda/bioconda-recipes/blob/master/recipes/pathogen-profiler/meta.yaml#L34

Doesn't really make a difference unless Jody plans on upgrading pathogen-profiler to use a higher delly version in the future (unsure how likely that is though...)

up to you if you want to change or not. I don't think it makes a difference right now

Reply:
nah, i think we're probably fine vfor now

Reply:
but i'll remove it so that future people don't automatically assume to use this version